frontend/apps/filemanager/filemanager.lua

```diff
@@ -70,10 +70,10 @@ function FileManager:onSetRotationMode(rotation)
     return true
 end
 
-function FileManager:setRotationMode()
+function FileManager:setRotationMode(init)
     local locked = G_reader_settings:readSetting("lock_rotation")
     local rotation_mode = G_reader_settings:readSetting("fm_rotation_mode") or 0
```

Comment:
Use the constant instead of 0, so I can get rid of that chunk in #6347 too ;p.

Reply:
done